ar_exercise/exercise.rb

```diff
@@ -34,38 +34,108 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.first or (Tenant.find(1))
+
 # get all tenants older than 65
+older_tenants = Tenant.where("age > 65")
+
 # get all apartments whose price is greater than $2300
+expensive_apartments = Apartment.where("monthly_rent > 2300", :rent)
+
+# color = :red
+# me = {"name" => "Adam", :age => 30}
+# me["name"]
+# me[:age]
+#
+# me = {:name => "Adam", :age => 30}
+# me = {name: "Adam", age: 30}
+
 # get the apartment with the address "6005 Damien Corners"
+damien_corners = Apartment.where(address: "6005 Damien Corners")
+
 # get all tenants in that apartment
+damien_corners.tenants
+
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+
+all_tenants.each do |tenant|
+  puts tenant.name
+  puts tenant.id
+end
+
 # Iterate over each apartment, for each apartment, display it's address and rent price
+all_apartments = Apartment.all
+all_apartments.each do |apartment|
+  puts apartment.address
+  puts appartment.monthly_rent
+  puts "*" * 50
+end
+
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+all_apartments.each do |apartment|
+  apartment.address
+  apartment.tenants.each do |tenant|
+    puts tenant.name
+  end
+end
 
 ################################################
 # CREATING / UPDATING / DELETING
 ################################################
 
-# Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
+# Hint, the following methods will help: `new`, `create`, `save`, `update`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+apartment1 = Apartment.create(address: "4128 Baldwin Crescent", monthly_rent: 7500, sqft: 2000, num_beds: 3, num_baths: 2.5)
+apartment2 = Apartment.create(address: "4055 Joshua Barney", monthly_rent: 5000, sqft: 1500, num_beds: 2, num_baths: 2)
+apartment3 = Apartment.create(address: "1211 Abraham Drive", monthly_rent: 10000, sqft: 3500, num_beds: 5, num_baths: 4)
+
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+new_tenant1 = Tenant.create(name: "Alexander	Hardy", age: "25", gender: "Male", apartment_id: 7)
```

Comment:
You shouldn't use hard coded IDs in a script, as they are subject to change (esp if you re-seed your DB, they may not start at 1). Instead do something like this:

```ruby
Tenant.create(name: "Alex", age: 21, gender: "female", apartment: apartment1)
# or this works too
Tenant.create(name: "Alex", age: 21, gender: "female", apartment_id: apartment1.id)
# or even this!
apartment1.tenants.create(name: "Alex", age: 21, gender: "female")
```